modules/pull/merge.go

```diff
@@ -100,13 +100,13 @@ func Merge(pr *models.PullRequest, doer *models.User, baseGitRepo *git.Repositor
 		return fmt.Errorf("git remote add [%s -> %s]: %s", headRepoPath, tmpBasePath, errbuf.String())
 	}
 
+	trackingBranch := "tracking"
```

Comment:
imho this will conflict if two PR are merged at same time

Reply:
Merging happens within an individual temporary repo:

https://github.com/zeripath/gitea/blob/499efda9d159bab311064d775ec1a2549342530b/modules/pull/merge.go#L56

This is simply an internal name.

Reply:
https://github.com/go-gitea/gitea/blob/ce8de3533485eed0c56059d6334a5031a73eed67/models/helper_directory.go#L29-L37

Reply:
But if there will be already branch named so?

Reply:
You're right. I've just pushed a cleaned version up.